pkg/driver/mount.go

```diff
@@ -177,21 +174,11 @@ func (pml *ProcMountLister) ListMounts() ([]mount.MountPoint, error) {
 		err    error
 	)
 
-	for i := 1; i <= procMountsReadMaxRetry; i += 1 {
-		mounts, err = os.ReadFile(pml.ProcMountPath)
-		if err == nil {
-			if i > 1 {
-				klog.V(4).Infof("Successfully read %s after %d retries", pml.ProcMountPath, i)
-			}
-			break
-		}
-
-		klog.Errorf("Failed to read %s on try %d: %v", pml.ProcMountPath, i, err)
-		time.Sleep(procMountsReadRetryBackoff)
-	}
+	mounts, err = os.ReadFile(pml.ProcMountPath)
 
 	if err != nil {
-		return nil, fmt.Errorf("Failed to read %s after %d tries: %w", pml.ProcMountPath, procMountsReadMaxRetry, err)
+		klog.Errorf("Failed to read %s on try %d: %v", pml.ProcMountPath, ProcMountsReadRetry, err)
+		ProcMountsReadRetry += 1
```

Comment:
This block basically ignores the error and returns an empty response. Although it won't fail, some operations depending on this result e.g., unmounting and skipping already mounted paths won't work.